test/unit/cart.js

```diff
@@ -300,23 +299,29 @@ describe('Cart class', () => {
   });
 
   describe('init', () => {
+    let superInitStub;
+    let fetchMoneyFormatStub;
+    beforeEach(() => {
+      superInitStub = sinon.stub(Component.prototype, 'init').resolves({model: {lineItems: [{id: 123, quantity: 5}]}});
+      fetchMoneyFormatStub = sinon.stub(cart, 'fetchMoneyFormat').returns(Promise.resolve('test'));
```

Comment:
Let's be consistent and use the `resolves()` function instead of the `returns(Promise....)`. 

Also, if we're not testing the resolved promise's return, let's leave it as an empty promise return.